manifest.php

```diff
@@ -24,10 +24,11 @@
     'label' => 'Browser and OS diagnostic tool',
     'description' => 'Check compatibility of the os and browser of a client',
     'license' => 'GPL-2.0',
-    'version' => '1.8.0',
+    'version' => '1.9.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
         'tao' => '>=4.0.0',
+        'taoClientRestrict' => '>=1.0.7',
```

Comment:
Unfortunately we can't afford to have this dependency. Most of instance with client-diag won't have client restrict installed